doc/howtos/backend/exercise-gantt

```diff
@@ -34,7 +34,7 @@ diff --git a/openacademy/models.py b/openacademy/models.py
 diff --git a/openacademy/views/openacademy.xml b/openacademy/views/openacademy.xml
 --- a/openacademy/views/openacademy.xml
 +++ b/openacademy/views/openacademy.xml
-@@ -143,11 +143,23 @@
+@@ -143,11 +143,22 @@
```

Comment:
I would guess that impacts pretty much every patch which follows (at least those which alter the openacademy.xml file), they should be updated.

Reply:
Ok, how can I fix this without breaking everything ^^

Reply:
Simplest is to load the series in quilt and replay it, refreshing patches when necessary. Though the series has been created with mq (which is significantly more annoying to set up though easier to work with when trying to actually create the series) which means there will be a fair amount of chaff to filter out.

Reply:
Well... if I just comment this line, that would fix it :D

Reply:
The other patches don't seem to alter this specific bit of the XML file so yeah that's an alternative.